test/images/scripts/lib/debian.install

```diff
@@ -46,6 +46,9 @@ if [ -n "$do_build" ]; then
     tar -xzf cockpit-*.tar.gz
     ( cd cockpit-*/
       cp -rp tools/debian debian
+      # Hack: Remove PCP build dependencies while pcp is not in testing
+      # (https://tracker.debian.org/pcp)
+      sed -i '/libpcp.*-dev/d' debian/control
```

Comment:
Thanks for the pointer. Indeed there are two `sed`s in there for mangling `debian/rules` already. If we want to generalize those, how about instead calling a `debian/rules prepare-dsc` target or a `debian/prepare-dsc.sh` from cockpituous, and keep the project specific bits in cockpit itself?

This should also get a `release` or `series` argument (passed from `release-{debian,ubuntu}`) so that the mangling can be release specific and `release-dsc` can actually put it into the changelog too (instead of `experimental`).